app/main/views/briefs.py

```diff
@@ -199,7 +199,7 @@ def view_response_result(brief_id):
     )
 
 
-def _render_not_eligible_for_brief_error_page(brief, clarification_question=False):
+def _render_not_eligible_for_brief_error_page(brief, clarification_question=False, custom_dimension=False):
```

Comment:
I suggest changing this extra parameter from the boolean into a string containing the reason the supplier isn't eligible, ie: `ineligibility_reason=None|'supplier-not-on-role'|'supplier-not-on-lot'|'supplier-not-on-framework'`.

I think doing this would mean the information passed through isn't only relevant to analytics and we could remove the block that sets the `dimension` variable in the template (line 7 in `app/templates/briefs/not_is_supplier_eligible_for_brief_error.html`).